### ISSUE TITLE
WebSocket connection error

### ISSUE THREAD

**Issue by craigsketchley** [rubenspgcavalcante/webpack-extension-reloader#86]
**Type:**
  - [x] bug
  - [ ] feature
  - [ ] enhancement
  - [ ] question

**Environment:**
  - OS: Ubuntu 18
  - Browser: Chrome 79
  - Library Version: 1.1.4

**I'm going to open a PR:**
  - [ ] yes
  - [x] no (well, not until we figure out the issue)

**Description:**
I am getting the following repeated error in the background page from what I assume is my background page trying to connect to `webpack-extension-reloader`.

```
WebSocket connection to 'ws://localhost:9090/' failed: Error in connection establishment: net::ERR_SSL_PROTOCOL_ERROR
```

I have reproduced it using the sample from this repo here: https://github.com/craigsketchley/webpack-extension-reloader-sample

Is anyone else getting this issue?

**Comment by craigsketchley** [comment]
Any thoughts on this @rubenspgcavalcante? I'm happy to look into this myself if you'd be happy to review a PR if it came to that.

**Comment by williamchatwin** [comment]
Any update on this? @craigsketchley have you found any solution to it?

**Comment by EgoPingvina** [comment]
I have this problem too.
@rubenspgcavalcante, could you pay attention to this problem, please.

**Comment by craigsketchley** [comment]
Yeah, it's unusable in it's current state for me. I've just uninstalled it - which is a shame because it's really useful. Is this still being maintained? Want me to look into it @rubenspgcavalcante?

**Comment by EgoPingvina** [comment]
@craigsketchley if you will find a solution or some another plugin for extensions hot-reload - say us please)
The author has not been active since December 13.

**Comment by ybentz** [comment]
The error you're getting is SSL related which is different than what I had but take a look at [this issue](#30) and the author's response, in case that's somehow related.
